rusoto/services/s3/src/generated.rs

```diff
@@ -18461,13 +18461,8 @@ impl<P, D> S3 for S3Client<P, D>
         let mut payload: Vec<u8>;
         payload = DeleteSerializer::serialize("Delete", &input.delete).into_bytes();
         let digest = md5::compute(&payload);
-        request.add_header("Content-MD5",
-                           &digest.to_base64(Config {
-                                                char_set: CharacterSet::Standard,
-                                                newline: Newline::LF,
-                                                pad: true,
-                                                line_length: None,
-                                            }));
+        // need to deref digest and then pass that reference:
```

Comment:
I think maybe this comment might be better in the code generator than repeatedly in the generated code?  Maybe?

Reply:
I prefer to have it as is: when I'm looking at the generated code when tracking down bugs, it explains the interesting collection of operators.  When I'm looking at codegen it serves the same purpose. 😄 

I *know* I'm going to look at that piece of generated code in six months and scratch my head as to why we're doing a deref followed by taking a reference. 😉